Extension/CloudFoundry/cloudfoundry.ts

```diff
@@ -6,16 +6,17 @@ import path = require('path');
 import fs = require('fs');
 import Q = require('q');
 
+tl.setResourcePath(path.join(__dirname, 'task.json'));
+
 var cfEndpoint = tl.getInput('cfEndpoint', true);
 var cfEndpointUrl = tl.getEndpointUrl(cfEndpoint, false);
 var cfEndpointAuth = tl.getEndpointAuthorization(cfEndpoint, false);
 var workingDir = tl.getInput('workingDirectory', true);
 
 var cfPath = tl.which('cf');
-var cfToolLocation = tl.getInput('cfToolLocation');
-if(cfToolLocation != tl.getVariable('System.DefaultWorkingDirectory')) {
-    //custom tool location for cf CLI was specified
-    cfPath = cfToolLocation;
+if(tl.filePathSupplied('cfToolLocation')) {
```

Comment:
p999: space after 'if'? :)

Reply:
I prefer that too, but I'd rather have it consistent with the rest of the file.

Reply:
You can fix the entire file by using Shift + Option +F in VSCode on Mac :)

Reply:
👍